static/js/marathon.js

```diff
@@ -130,11 +130,15 @@ let app = new Vue({
 
         async pageCallback(page, loadTime) {
 
-            // console.log("page callback")
-
-            this.clicksRemaining -= 1;
+            if (this.path.length == 0) {
+                this.path.push(page);
+                this.clicksRemaining -= 1;
+            } else if (this.path[this.path.length - 1] != page) {
```

Comment:
Why not just use an or and short circuiting for this?

Reply:
forgot that was a thing lmao, fixed